xml/System.IO.Pipelines/PipeReader.xml

```diff
@@ -97,8 +98,12 @@ The memory for the consumed data will be released and no longer available.
 
 The <xref:System.IO.Pipelines.ReadResult.Buffer> previously returned from <xref:System.IO.Pipelines.PipeReader.ReadAsync(System.Threading.CancellationToken)> must not be accessed after this call.
 
+This call is equivalent to calling <xref:System.IO.Pipelines.AdvanceTo(System.SequencePosition, System.SequencePosition)> with the examined position same as the consumed position.
```

Comment:
Nit:

```suggestion
This is equivalent to calling <xref:System.IO.Pipelines.AdvanceTo(System.SequencePosition, System.SequencePosition)> with identical examined and consumed positions.
```

Reply:
the corrections make it more concise and clear.